Dockerfile

```
@@ -18,7 +18,7 @@
 # FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
 # IN THE SOFTWARE.
 
-FROM python:3.11.0-alpine3.17
+FROM python:3.11.3-alpine3.18
```

We can just use `major.minor` imo:

```suggestion
FROM python:3.11-alpine3.18
```